docs/hoc/modelspec/programmatic/mechanisms/mech.rst

```diff
@@ -826,7 +826,7 @@ Mechanisms
 
 **fastpas**
 
-        See `<nrn src dir>/src/nrnoc/passive0.c <http://neuron.yale.edu/hg/neuron/nrn/file/tip/src/nrnoc/passive0.c>`_
+        See `<nrn src dir>/src/nrnoc/passive0.c <https://github.com/neuronsimulator/nrn/blob/master/src/nrnoc/passive0.c>`_
```

Comment:
```suggestion
        See `<nrn src dir>/src/nrnoc/passive0.c <https://github.com/neuronsimulator/nrn/blob/master/src/nrnoc/passive0.cpp>`_
```

Reply:
latest version fixes the links and the displayed text